mlflow/keras.py

```diff
@@ -144,24 +158,42 @@ def log_model(keras_model, artifact_path, conda_env=None, **kwargs):
     >>>   mlflow.keras.log_model(keras_model, "models")
     """
     Model.log(artifact_path=artifact_path, flavor=mlflow.keras,
-              keras_model=keras_model, conda_env=conda_env, **kwargs)
+              keras_model=keras_model, conda_env=conda_env, custom_objects=custom_objects, **kwargs)
+
 
+def _save_custom_objects(path, custom_objects):
```

Comment:
Can we document the semantics of `path`? Does `path` refer to the MLflow model path or the data directory path or the destination path for the pickled custom objects?

Perhaps it would be clearer if this method accepted the destination path for the custom objects directly; the caller can then construct the path and call `_save_custom_objects(os.path.join(path, custom_objects_subpath), ...)`.

Reply:
Took a pass at it without that - let me know if you still think it's necessary with the documentation